packages/tools/pie-icons-webc/tests/icon.test.js

```diff
@@ -1,30 +1,68 @@
 import { html, fixture, expect } from '@open-wc/testing';
 import { sizeToValueMap, largeIconSizeDefault, regularIconSizeDefault } from '@justeattakeaway/pie-icons-configs';
-import { IconAlcoholFilled } from '../icons/IconAlcoholFilled';
-import { IconAlcoholFilledLarge } from '../icons/IconAlcoholFilledLarge';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { IconAlcoholFilled, IconAlcoholFilledLarge } from '../dist/icons';
```

Comment:
Since all icon files have been removed, does this mean we need to run an icons build before we can run unit tests?
If so, we should either document this, or make sure the dist build is triggering before running the tests

Reply:
Our `turbo.json` handles the dependency here, so running the tests will trigger the build first 👍🏻